tests/acceptance/features/webUICreateFilesFolders/createFolders.feature

```diff
@@ -67,3 +67,27 @@ Feature: create folders
     Then folder "sub-folder" should be listed on the webUI
     When the user reloads the current page of the webUI
     Then folder "sub-folder" should be listed on the webUI
+
+
+  Scenario: Create folders with dot in the name
+    When the user creates the following folders using the webUI
+      | folders  |
+      | fo.      |
+      | fo.1     |
+      | fo...1.. |
+      | ...      |
+      | ..fo     |
+      | fo.xyz   |
+    Then the following folders should be listed on the webUI
+      | folders  |
+      | fo.      |
+      | fo.1     |
+      | fo...1.. |
+      | ...      |
+      | ..fo     |
+      | fo.xyz   |
+
+
+#    Scenario: Try to create a folder with .part in name
+#      When the user tries to create a folder with the invalid name "folder.part" using the webUI
+#      Then the create folder button should be disabled
```

Comment:
left overs?

Reply:
I try I fail. Any suggestion.

Reply:
why does the test fail? there might be an issue. please look at the failing causes more carefully.

Reply:
https://github.com/owncloud/web/issues/5518

Reply:
You can convert this into a bug demostration... Tag the scenario with the created issue